src/collective/volto/formsupport/datamanager/catalog.py

```diff
@@ -110,6 +130,13 @@ def search(self, query=None):
             )
         return records
 
+    def count(self, query=None):
+        records = []
+        if not query:
+            records = self.soup.data.values()
+
+        return len(records)
+
```

Comment:
ok, questo ti ritorna il numero di entry. non capisco come lo colleghi al form... mi pare che così cerchi in tutto il soup

Reply:
onestamente avevo anche io quel dubbio, ma mi sono basato sulla search per farlo, e la search funziona

Reply:
@eikichi18 IMHO this is a bug. Try adding 2 modules, each with limit 2, add one record to each, try adding a second record. Does this work?